src/main/scala/mastermind/Mastermind.scala

```diff
@@ -13,19 +13,24 @@ val validColors = List('B', 'Y', 'R', 'G')
 
 /** Get a random color from the list of valid colors */
 def getRandomColor(): Color =
-  ???
+  validColors(Random.between(0, validColors.length))
 
 /** Given four colors, make a board from them */
 def makeBoardFromColors(c1: Color, c2: Color, c3: Color, c4: Color): Board =
-  ???
+  List(c1, c2, c3, c4).mkString
```

Comment:
I did this using s and '$' notation instead, but I think your solution is more readable!